jib-cli/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - Downgraded Google HTTP libraries to 1.34.0 to resolve network issues. ([#3415](https://github.com/GoogleContainerTools/jib/pull/3415), [#3058](https://github.com/GoogleContainerTools/jib/issues/3058), [#3409](https://github.com/GoogleContainerTools/jib/issues/3409))
+- Changed the default base image of the Jib CLI `jar` command from the `adoptopenjdk` images to the [`eclipse-temurin`](https://hub.docker.com/_/eclipse-temurin) (for Java 8 and 11) and [`azul/zulu-openjdk`](https://hub.docker.com/r/azul/zulu-openjdk) (for Java 17) images on Docker Hub. Note that Temurin (by Adoptium) is the new name of AdoptOpenJDK. ([#3491](https://github.com/GoogleContainerTools/jib/pull/3491))
```

Comment:
Optional: should we also consider adding integration testing for a java 17 jar in [JarCommandTest](https://github.com/GoogleContainerTools/jib/blob/4e5d44c5aead88b2836b41371cee57d6f70f1c2b/jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/JarCommandTest.java) or for the maven or gradle plugin?

Reply:
So I added a Java 17 test project for the Gradle plugin, although it won't get utilized until we set up Java 17 on Kokoro. But once we have that, the test will kick in.